codec-mqtt/src/test/java/io/netty/handler/codec/mqtt/MqttReasonCodesTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.codec.mqtt;
+
+import org.junit.jupiter.api.Test;
+
+import static io.netty.handler.codec.mqtt.MqttReasonCodes.Disconnect;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MqttReasonCodesTest {
```

Comment:
Please add more test cases

Reply:
Do you mean to add one test for each new enum added?

Reply:
Not every, but a couple of will do.

Reply:
Also, try simulating the overflow warning of LGTM. Make sure it does not actually happen.

Reply:
I don't think it'll be that much work to just iterate the enums and check conversion of all the values. Would certainly be less work than simulating some aspect of the protocol.

Reply:
Fixed with 8ac7d530e764348efeb69f24b985027a29d44589